packages/caver-klay/caver-klay-accounts/src/index.d.ts

```diff
@@ -0,0 +1,162 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import RpcCallToMethod from '../../../caver-rtm/src'
+import { SignatureObject } from '../../../caver-utils/src/utils'
+import { EncryptedKeyOptions, Keystore } from '../../../caver-wallet/src/keyring/keyringHelper'
+import Account, { AccountKey } from './account/account'
+import AccountForUpdate from './account/accountForUpdate'
+import AccountKeyMultiSig from './accountKey/accountKeyMultiSig'
+import AccountKeyPublic from './accountKey/accountKeyPublic'
+import AccountKeyRoleBased, { RoleBasedKeyObject } from './accountKey/accountKeyRoleBased'
+
+export interface TransactionObject {
+    feePayerSignatures?: string[][]
+    message?: string
+    messageHash: string
+    r: string
+    rawTransaction: string
+    s: string
+    senderTxHash: string
+    signatures?: string[]
+    signature?: string
+    txHash: string
+    v: string
+}
+
+export interface PublicKeyObject {
```

Comment:
```suggestion
export interface RoleBasedKeyObject {
```

Reply:
Is it only for RoleBasedKey?
It written based on the test `caver.klay.accounts.js` line 4190-4209, 4248-4261, 4263-4277.

I think, `accountKeyToPublicKey` function takes as input the keys you want returned as publicKey.
So I don't think RoleBasedKeyObject is appropriate. What about you?

Reply:
If you see https://docs.klaytn.com/bapp/sdk/caver-js/v1.4.1/api-references/caver.klay.accounts#accountkeytopublickey here, you can see the difference between accountKey types.
If the keys are defined for each role, it means AccountKeyRoleBased.

Reply:
Can I replace `PublicKeyObject` with `RoleBasedKeyObject` in `accountKeyRoleBased.d.ts`?
I think it's duplicated one.

Reply:
I think you can do that !

Reply:
Fixed. please check aa2ddcf